presto-native-execution/presto_cpp/presto_protocol/presto_protocol.h

```diff
@@ -3313,3 +3313,11 @@ enum class NodeState { ACTIVE, INACTIVE, SHUTTING_DOWN };
 extern void to_json(json& j, const NodeState& e);
 extern void from_json(const json& j, NodeState& e);
 } // namespace facebook::presto::protocol
+namespace facebook::presto::protocol {
+struct ConvertToVeloxPlanResponse {
```

Comment:
I defined this simple message to properly send the error information in the response